src/lamplib/src/genny/toolchain.py

```diff
@@ -199,7 +175,9 @@ class ToolchainDownloader(Downloader):
     # If we were 💅 we could do the string logic here in python, but we're not that fancy.
     #
 
-    TOOLCHAIN_BUILD_ID = "cf5743e0a96212cb7fae298399b1759489723cdb_24_04_16_17_45_56"
+    TOOLCHAIN_BUILD_ID = (
+        "patch_cf5743e0a96212cb7fae298399b1759489723cdb_664d957bd979e30007b58a69_24_05_22_06_49_32"
```

Comment:
I assume this changes to a non-patch id before the final merge?

Reply:
Yep, the PR in vcpkg needs to get merged so I have a real ID. After that I'll update it, get green on these tests, update the mongo patch tests and get green on those.

I'll be specifically checking that Genny stops trying to download the same toolchain several times once it has a real ID that it can compare to the git rev-parse it performs on the toolchain dir.